web/src/reducers/apd.selectors.js

```diff
@@ -128,3 +128,12 @@ export const selectApdDashboard = createSelector([selectApds], ({ byId }) =>
 );
 
 export const selectLastSavedTimestamp = state => state.apd.data.updated;
+
+export const getYearsAlert = ({
```

Comment:
Doesn't look like this is used anywhere

Reply:
Nice catch.  This was a different approach I tried and abandoned as a bad idea.